### PR TITLE
set customActiveChannel

### PR DESCRIPTION
**Pull request by jaapbakker88**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/ChannelList.js [hunk 1]
```diff
@@ -213,6 +218,17 @@ class ChannelList extends PureComponent {
           refreshing: false,
         };
       });
+
+      // Set a channel as active and move it to the top of the list.
+      if (this.props.customActiveChannel) {
```
Comment (vishalnarkhede) [inline src/components/ChannelList.js]: Can we move this logic to line 202? That way we will only make call to `setActiveChannel` once

Comment (jaapbakker88) [reply src/components/ChannelList.js]: Yeah, the problem is we don't have the channels in state yet over there.

Comment (vishalnarkhede) [reply src/components/ChannelList.js]: ah ok!!

**Comment by ludovicthomas** [comment]
@vishalnarkhede it seems that if the `customActiveChannel` contains a `channelId` that is not loaded from the channels response, it will not load it. Is it a known issue? Thanks for your feedback.

**Pull request merged**